src/main/java/seedu/address/logic/parser/SocialInfoMapping.java

```diff
@@ -45,6 +45,18 @@ public static SocialInfo parseSocialInfo(String rawSocialInfo) throws IllegalVal
 
     }
 
+    //@@author sarahnzx
+    public static String getSocialType(String socialType) throws IllegalValueException {
```

Comment:
Just for future reference, it might be better to have this function return an optional, so you don't have to do the complicated test on parseSelect